### PR TITLE
add streamlit app example

### PR DESCRIPTION
**Pull request by murilommen**
this PR adds a package to a new directory called `examples` with a chatbot app developed in streamlit that uses a deployed langkit-container's endpoint to instrument `openllmtelemetry` and demonstrates prompt/response guardrailing with both WhyLabs Secure tools.

### CODE REVIEW

File: examples/streamlit-app/app.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    if "openai_model" not in st.session_state:
+        st.session_state["openai_model"] = "gpt-3.5-turbo"
+
+    # Initialize chat history
+    if "messages" not in st.session_state:
+        st.session_state.messages = []
+    if "context" not in st.session_state:
+        st.session_state.context = [{"role": "system",
+                                     "content": """
+                                You are a helpful customer service chatbot. You strive to make customer happy. Here are
```
Comment (andyndang) [inline examples/streamlit-app/app.py]: suggestion: make this a `system-message.txt` so user can change it easily wthout the code?

File: examples/streamlit-app/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+streamlit==1.31.0
+openllmtelemetry[openai]==0.0.1b15
```
Comment (andyndang) [inline examples/streamlit-app/requirements.txt]: do we want to version lock? Thoughts on this @jamie256 ?

**Review by andyndang — COMMENTED** [review]
Couple of comments

File: examples/streamlit-app/requirements.txt (hunk 2 above)
Comment (jamie256) [reply examples/streamlit-app/requirements.txt]: It should be fine to allow >=0.0.1b15

Also suggest using both bedrock and openai extras

**Review by andyndang — APPROVED** [review]
LGTM

**Pull request merged**